README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started
+
+Here's how to get the dev environment setup for our Prysmatic Website:
+1. Via terminal/command line cd into the project's main directory
+2. Run `npm install`
+3. Run `npm start`
+
+## Available Commands
+
+| Command   | Description                                                                                                                                                               |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| npm start | Builds the Prysmatic website and opens it in your default browser                                                                                                         |
+| npm build | Builds the Prysmatic website                                                                                                                                              |
+| npm test  | Sets up a test environment for development (more logging and such)                                                                                                        |
+| npm eject | If you aren’t satisfied with the build tool and configuration choices, you can eject at any time. This command will remove the single build dependency from your project. |
+
+# Become a Shardler
```

Comment:
Also a good addition to have a readme, but I think this can be condensed down a bit.  User shouldn't need to build since Travis will handle that. We could probably condense this down to 'Getting Started' and 'Looking to Contribute?'